vault/token_store.go

```diff
@@ -295,24 +286,21 @@ func NewTokenStore(c *Core, config *logical.BackendConfig) (*TokenStore, error)
 				},
 
 				Callbacks: map[logical.Operation]framework.OperationFunc{
-					logical.ReadOperation: t.handleLookupSelf,
+					logical.ReadOperation:   t.handleLookupSelf,
```

Comment:
We can but I left it in for backwards compatibility, as it's not a security risk.